src/coreclr/vm/multicorejitimpl.h

```diff
@@ -45,7 +46,7 @@ const int      MAX_WALKBACK      = 128;
 
 enum
 {
-    MULTICOREJIT_PROFILE_VERSION   = 102,
+    MULTICOREJIT_PROFILE_VERSION   = 103,
```

Comment:
This shouldn't be necessary if the `NO_JIT_TAG` flag is removed (based on my other comment), if I understood correctly that was the only change to the storage format